metaflow/plugins/aws/batch/batch_decorator.py

```diff
@@ -37,51 +37,51 @@ class BatchDecorator(StepDecorator):
 
     Parameters
     ----------
-    cpu : int, default: 1
+    cpu : int, default 1
```

Comment:
style: trying to be consistent and remove : to just have `default <value>`. Either way works. Just consistency.